keps/sig-node/3695-pod-resources-for-dra/kep.yaml

```diff
@@ -0,0 +1,56 @@
+title: Extend the PodResources API to include resources allocated by DRA
+kep-number: 3695
+authors:
+  - "@moshe010"
+owning-sig: sig-node
+participating-sigs: []
+status: provisional
+creation-date: implementable
+reviewers:
+  - "@ffromani"
+  - "@swatisehgal"
+approvers:
+  - "@sig-node-leads"
+see-also:
+  - "keps/sig-node/2403-pod-resources-allocatable-resources"
+  - "keps/sig-node/606-compute-device-assignment/"
+  - "keps/sig-node/2043-pod-resource-concrete-assigments/"
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.27"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.27"
+  beta: "v1.30"
+  stable: "v1.32"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: DynamicResourceAllocation
+    components:
+      - kube-apiserver
+      - kube-controller-manager
+      - kube-scheduler
+      - kubelet
```

Comment:
From what I understand you should only be making changes to the `kubelet`, right?

Let's wait to see what the PRR reviewers say, but I would imagine you only need to list the `kubelet` here since that is the only component your KEP will be modifying that uses this feature gate (even if it exists for other purposes in these other components).

Reply:
yes, I added the other because they required for DRA to function when you enable DynamicResourceAllocation feature gate.

Reply:
I guess we need to differentiate this proposal from DRA carefully. DRA could be treated as a dependency instead of playing directly here? (unless you plan to touch DRA behavior with this KEP which shouldn't happen^^) Curious about what @johnbelamaric thinks?

Reply:
I think this is OK. The metadata here isn't expressive enough to capture it perfectly, but it is true these all have to be on in these components, so it's useful to document it here as well as in the text above. The meta data field here is for generating docs (not sure if it's used yet though by the docs team).